packages/xo-web/src/xo-app/xoa/support/index.js

```diff
@@ -22,35 +26,83 @@ const Support = decorate([
+    computed: {
+      stdoutSupportTunnel: (_, { tunnelState }) =>
+        tunnelState === undefined
+          ? undefined
+          : { __html: marked(tunnelState.stdout) },
+    },
   }),
   injectState,
-  ({ effects, state: { stdoutCheckXoa } }) => (
+  ({
+    effects,
+    state: { stdoutCheckXoa, stdoutSupportTunnel },
+    tunnelState: { open, stdout } = { open: false, stdout: '' },
+  }) => (
     <Container>
+      {COMMUNITY && (
+        <Row className='mb-2'>
+          <Col>
+            <span className='text-info'>{_('supportCommunity')}</span>
+          </Col>
+        </Row>
+      )}
       <Row>
         <Col mediumSize={6}>
           <Card>
             <CardHeader>{_('xoaCheck')}</CardHeader>
-            {COMMUNITY ? (
-              <CardBlock>
-                <span className='text-info'>{_('checkXoaCommunity')}</span>
-              </CardBlock>
-            ) : (
-              <CardBlock>
-                <ActionButton
-                  btnStyle='success'
-                  handler={effects.checkXoa}
-                  icon='diagnosis'
-                >
-                  {_('checkXoa')}
-                </ActionButton>
-                <hr />
-                <pre
-                  dangerouslySetInnerHTML={{
-                    __html: ansiUp.ansi_to_html(stdoutCheckXoa),
-                  }}
-                />
-              </CardBlock>
-            )}
+            <CardBlock>
+              <ActionButton
+                btnStyle='success'
+                disabled={COMMUNITY}
+                handler={effects.checkXoa}
+                icon='diagnosis'
+              >
+                {_('checkXoa')}
+              </ActionButton>
+              <hr />
+              <pre
+                dangerouslySetInnerHTML={{
+                  __html: ansiUp.ansi_to_html(stdoutCheckXoa),
+                }}
+              />
+            </CardBlock>
+          </Card>
+        </Col>
+        <Col mediumSize={6}>
+          <Card>
+            <CardHeader>{_('supportTunnel')}</CardHeader>
+            <CardBlock>
+              <Row>
+                <Col>
+                  {open ? (
+                    <ActionButton
+                      btnStyle='primary'
+                      disabled={COMMUNITY}
+                      handler={closeTunnel}
+                      icon='remove'
+                    >
+                      {_('closeTunnel')}
+                    </ActionButton>
+                  ) : (
+                    <ActionButton
+                      btnStyle='success'
+                      disabled={COMMUNITY}
+                      handler={openTunnel}
+                      icon='open-tunnel'
+                    >
+                      {_('openTunnel')}
+                    </ActionButton>
+                  )}
+                </Col>
+              </Row>
+              <hr />
+              {open || stdout !== '' ? (
+                <pre dangerouslySetInnerHTML={stdoutSupportTunnel} />
```

Comment:
Just to make it a little bit clearer:
```suggestion
                <pre className={!open && stdout !== '' && 'text-danger' } dangerouslySetInnerHTML={stdoutSupportTunnel} />
```